specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -2408,49 +2616,81 @@
             "required": true,
             "type": "string"
           },
+          {
+            "name": "assemblyArtifactName",
+            "description": "The assembly artifact name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
           {
             "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/WorkflowTriggerCallbackUrl"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/batchConfigurations": {
+      "get": {
+        "tags": [
+          "IntegrationAccountBatchConfigurations"
+        ],
+        "operationId": "IntegrationAccountBatchConfigurations_List",
+        "description": "List the batch configurations for an integration account.",
+        "x-ms-examples": {
```

Comment:
Is there a reason this one isn't x-ms-pageable?

Reply:
I was under the impression that x-ms-pageable required a page size query param. Now that I know that it doesn't, there is no reason. I'll add it